index.html

```diff
@@ -332,10 +362,10 @@ <h3>
             </svg>
 
 
-            <textarea id="subject" name="subject" required>Enter Text Here</textarea>
-            <button type="submit">Submit</button><br>
+            <textarea id="subject" name="subject" required>Hello Marianne! Good to see you!</textarea>
+            <button type="submit">Get in touch</button><br>
           </form>
-      </div>
+        </div>
       </section>
 
 
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you delete all unnecessary empty code lines to make your codebase looks professional.
- [ ] Kindly make sure you add the footer with social media icons as it misses on your project to stick to the Figma template. And once you added it make sure you style it correctly like on The Figma template.